server/routes/user.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+
+const User = require('../models/user');
+const auth = require('../middleware/auth');
+
+const router = express.Router();
+
+/////// User sign up route handler ///////
+router.post('/api/users/signup', async (req, res) => {
+	const user = new User(req.body);
+
+	try {
+		await user.save();
+		const token = await user.generateAuthToken();
+		res.cookie('accessToken', token, { httpOnly: true });
+		res.status(201).send(user);
+	} catch (e) {
+		res.status(400).send(e);
+	}
+});
+
+/////// User login route handler ///////
+router.post('/api/users/login', async (req, res) => {
+	const { email, password } = req.body;
+	try {
+		const user = await User.findUserByCredentials(email, password);
+		const token = await user.generateAuthToken();
+		res.cookie('accessToken', token, { httpOnly: true });
```

Comment:
You can use .clearCookie here
https://expressjs.com/en/api.html#res.clearCookie